### DISCUSSION TITLE
From version 2.25 index.html file is not generated

### DISCUSSION THREAD

**Discussion by wdcs-tanishqakotiya** [allure-framework/allure2#2271] [Questions & Support] [answered]
I have been using allure version 2.23 from long time and all my allure reports were being generated but as soon as I upgrade allure to version 2.25 whenever I generate the allure report my index.html and other js files are not generated. 
What is the resolution of that?

**Comment by baev** [comment]
Could you please specify some additional details on your problem?

**Comment by wdcs-tanishqakotiya** [comment]
Sure @baev 
Since I have updated the allure version from 2.23 to 2.25 I am unable to generate or serve my allure reports
For instance, I am sharing the response thrown on serving the allure report from the allure-result directory

![Screenshot 2024-01-09 at 12 54 27 PM](https://github.com/allure-framework/allure2/assets/116056174/314d4832-47ed-4972-b178-353cddeb7191)


I am not sure what could be the possible change, I have been following the same process and steps that I was following when working with an older version of Allure

**Comment by baev** [comment, marked as answer]
Issue https://github.com/allure-framework/allure2/issues/2283
Fixed by https://github.com/allure-framework/allure2/pull/2284

will be released later this week

**Reply by baev** [reply]
Released as [2.26.0](https://github.com/allure-framework/allure2/releases/tag/2.26.0)

**Comment by wdcs-tanishqakotiya** [comment]
Thanks @baev 
reports are generated now from v 2.26
